code/drasil-lang/lib/Language/Drasil/Expr/Lang.hs

```diff
@@ -1,6 +1,8 @@
 {-# LANGUAGE GADTs                 #-}
 {-# LANGUAGE InstanceSigs          #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-overlapping-patterns #-}
```

Comment:
When HLS suggests these kinds of things, it's usually a hint that something is going awry (for the majority of our of Haskell use, at least). I think we can remove these two pragmas.